### ISSUE TITLE
ruamel issue: AttributeError: 'NoneType' object has no attribute 'anchor'

### ISSUE THREAD

**Issue by alexcg1** [jina-ai/jina#1250]
**Describe your problem**
1. Installed Jina in virtualenv
2. Ran `jina hello-world`
3. After downloading (I think it finished) fashion-mnist it immediately dies. Output here (trimmed out Jina banner and download status of mnist)
[log.txt](https://github.com/jina-ai/jina/files/5511298/log.txt)

**What is your guess?**
Some issue with ruamel

**Environment**

```
jina                          0.7.7
jina-proto                    0.0.75
jina-vcs-tag                  (unset)
libzmq                        4.3.2
pyzmq                         1.19.4
protobuf                      3.13.0
proto-backend                 python
grpcio                        1.33.2
ruamel.yaml                   0.16.12
python                        3.9.0
platform                      Linux
platform-release              5.8.18-300.fc33.x86_64
platform-version              #1 SMP Mon Nov 2 19:09:05 UTC 2020
architecture                  x86_64
processor                     x86_64
jina-resources                /home/alexcg/work/code/jupyter/env/lib/python3.9/site-packages/jina/resources
JINA_ARRAY_QUANT              (unset)
JINA_BINARY_DELIMITER         (unset)
JINA_CONTRIB_MODULE           (unset)
JINA_CONTRIB_MODULE_IS_LOADING(unset)
JINA_CONTROL_PORT             (unset)
JINA_DB_COLLECTION            (unset)
JINA_DB_HOSTNAME              (unset)
JINA_DB_NAME                  (unset)
JINA_DB_PASSWORD              (unset)
JINA_DB_USERNAME              (unset)
JINA_DEFAULT_HOST             (unset)
JINA_DISABLE_UVLOOP           (unset)
JINA_EXECUTOR_WORKDIR         (unset)
JINA_FULL_CLI                 (unset)
JINA_IPC_SOCK_TMP             (unset)
JINA_LOG_CONFIG               (unset)
JINA_LOG_NO_COLOR             (unset)
JINA_POD_NAME                 (unset)
JINA_PROFILING                (unset)
JINA_RANDOM_PORTS             (unset)
JINA_SOCKET_HWM               (unset)
JINA_TEST_GPU                 (unset)
JINA_TEST_PRETRAINED          (unset)
JINA_VCS_VERSION              (unset)
JINA_WARN_UNNAMED             (unset)
```

**Comment by cristianmtr** [comment]
I see you're using `python3.9`. AFAIK, Jina doesn't explicitly support that version

**Comment by alexcg1** [comment]
Fixed by using Python 3.7
